addons/l10n_dk_oioubl/data/oioubl_templates.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="l10n_dk_oioubl.oioubl_PaymentTermsType" inherit_id="account_edi_ubl_cii.ubl_20_PaymentTermsType">
+        <xpath expr="//*[local-name()='Amount']" position="after">
+            <t xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
+               xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
+                <t t-set="settlement_period" t-value="vals.get('settlement_period')"/>
+                <cac:SettlementPeriod>
+                    <cbc:StartDate><t t-out="settlement_period.get('start_date')"/></cbc:StartDate>
+                    <cbc:EndDate><t t-out="settlement_period.get('end_date')"/></cbc:EndDate>
+                </cac:SettlementPeriod>
```

Comment:
```suggestion
                <t t-set="settlement_period" t-value="vals['settlement_period']"/>
                <cac:SettlementPeriod>
                    <cbc:StartDate><t t-out="settlement_period['start_date']"/></cbc:StartDate>
                    <cbc:EndDate><t t-out="settlement_period['end_date']"/></cbc:EndDate>
                </cac:SettlementPeriod>
```
... since you ensure the value is always there python-side

Reply:
Regarding the details of PaymentTermsType on:
http://www.datypic.com/sc/ubl21/t-cac_PaymentTermsType.html

I think we could also specify `PaymentDueDate`. 

Also, seing `SettlementDiscountAmount`, I think we should add information about early payment discount in here but I'm not sure :/